src/index.html

```diff
@@ -11,7 +11,103 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form method="post">
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <label class="box">
+          Surname:
+          <input type="text" name="attribute"
+          autocomplete="off"
+          >
+        </label>
+        <label class="box">
+          Name:
+          <input type="text" name="attribute"
+          autocomplete="off"
+          >
+        </label>
+        <label class="box">
+          How old are You?
+          <input type="number" name="attribute"
+          value="12" min="1"
+          max="100"
+          >
+        </label>
+        <label class="box">
+          Full date of birth:
+          <input type="date" name="attribute">
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input type="checkbox" name="attribute"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label class="box">
+          E-mail:
+          <input
+            type="email"
+            name="attribute"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password: <input type="password" name="attribute">
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <label class="box">
+          Do you love cats?
+          <input type="radio" name="attribute"> Yes
+          <input type="radio" name="attribute"> No
+        </label>
+        <label class="box">
```

Comment:
```suggestion
          Do you love cats?
          <input type="radio" name="attribute"> 
            Yes
          <input type="radio" name="attribute"> 
            No
        </label>
        <label class="box">
```

варто огорнути інпути в label, щоб при натисканні на слово No вибиралась опція No

https://user-images.githubusercontent.com/102413088/189646042-6c22dc30-c4ec-4626-b7a1-4334a1ed58a3.mov